src/main/java/com/gmail/nossr50/skills/woodcutting/WoodcuttingManager.java

```diff
@@ -317,8 +328,9 @@ private void dropTreeFellerLootFromBlocks(Set<BlockState> treeFellerBlocks) {
     }
 
     private int updateProcessedLogCount(int xp, int processedLogCount, int beforeXP) {
-        if(beforeXP != xp)
+        if (beforeXP != xp) {
```

Comment:
Try to avoid changing the style
 (I know mcMMO code style isn't consistent, its made more consistent in another branch)